packages/next-sitemap/src/url/create-url-set/index.ts

```diff
@@ -118,7 +118,9 @@ export const createUrlSet = async (
     ...x,
     loc: absoluteUrl(config.siteUrl, x.loc, config.trailingSlash), // create absolute urls based on sitemap fields
     alternateRefs: (x.alternateRefs ?? []).map((alternateRef) => ({
-      href: absoluteUrl(alternateRef.href, x.loc, config.trailingSlash),
```

Comment:
This needs to land, I am not sure how this was intended to work initially but joining two potentially localized paths is obviously problematic... There really needs to be a way to apply this absolute href or tell the system to ignore the initial path segment if it's localized. Referencing x.loc directly without stripping the localized leading path segment will lead to things like https://www.example.com/es-mx/pt-br/page.html when providing a fully localized path map.

@iamvishnusankar are you cool landing this? Good escape-hatch IMO

Reply:
Thanks!! And sorry for my whiney post... :smh: I obviously coulda used a Snickers™️